src/thor/route_action.cc

```diff
@@ -276,14 +287,23 @@ thor::PathAlgorithm* thor_worker_t::get_path_algorithm(const std::string& routet
     for (auto& edge2 : destination.path_edges()) {
       if (edge1.graph_id() == edge2.graph_id() ||
           reader->AreEdgesConnected(GraphId(edge1.graph_id()), GraphId(edge2.graph_id()))) {
+
+        // We need time dependence so we use timedep_foward when its trivial (should be short so not a
+        // real issue that its not invariant)
+        if (options.has_date_time_type() && options.date_time_type() == Options::invariant) {
```

Comment:
in the case that its a trivial route then we do let `timedep_forward` handle invariant as i didnt want to modify `astar` to do time dependence for just this tiny use-case